test/cpp/test_aten_xla_tensor_1.cpp

```diff
@@ -2033,6 +2033,34 @@ TEST_F(AtenXlaTensorTest, TestScatterReduceMaxInPlace) {
   ExpectCounterChanged("xla::scatter_reduce", cpp_test::GetIgnoredCounters());
 }
 
+TEST_F(AtenXlaTensorTest, TestUnsafeIndex) {
```

Comment:
Is it possible / do we want to merge this test and that from `aten.index`?

Reply:
@JackCaoG @wonjoolee95 @lezcano 
I guess it's possible. But I'd argue against it for consistency. But, wouldn't mind doing this. What do you think?

Reply:
Both sounds okay to me, but this is fine as it is now given the structure of our cpp unit tests.